mkdocs.yml

```diff
@@ -93,38 +96,27 @@ plugins:
     search: {}
     gen-files:
         scripts:
-            - docs/collections/generate_catalog.py
+            - docs/integrations/generate_catalog.py
             - docs/recipes/generate_catalog.py
 watch:
     - mkdocs.yml
     - mkdocs.insiders.yml
     - src/prefect/
 nav:
-    # - Prefect 2: index.md
-    - Getting Started:
-        - Welcome to Prefect 2: index.md
-        - Quick Start: getting-started/overview.md
+    - Getting Started: 
+        - index.md  
         - Installation: getting-started/installation.md
-        - Tutorials:
-            - Prefect Fundamentals:
-                - tutorials/first-steps.md
-                - tutorials/flow-task-config.md
-                - tutorials/execution.md
-                - tutorials/orchestration.md
-                - tutorials/deployments.md
-                - tutorials/storage.md
-                - tutorials/docker.md
-                - Prefect Cloud Quickstart: ui/cloud-quickstart.md
-                - Cloud Agent Quickstart: tutorials/cloud-agent.md
-            - Advanced Tutorials:
-                - tutorials/testing.md
-                - tutorials/dask-ray-task-runners.md
-        - Migration Guide: migration-guide.md
-        - Prefect Recipes: 
-            - Recipes Catalog: recipes/recipes.md
-            - Contributing Recipes: recipes/contribute-recipes.md
-    - Concepts:
-        - Overview: concepts/overview.md
+        - Cloud Quickstart: ui/cloud-quickstart.md
+    - Tutorials: 
```

Comment:
```suggestion
    - Tutorial: 
```

Reply:
Right now it's still a bunch of tutorials, right? So do you think we should leave it as the plural until we condense?

Reply:
Well I guess it's semantic, but to me it's one tutorial... _the tutorial_ on how to use Prefect.